world/dominion/plots/models.py

```diff
@@ -1475,7 +1483,7 @@ def check_action_against_maximum_allowed(self):
         if self.org:
             action_ids = (
                 self.org.actions.exclude(id=self.id)
-                .filter(Q(beat__isnull=True) | Q(beat__episode=episode))
+                .filter(Q(episode=episode))
```

Comment:
This will remove the previous condition so that they no longer apply at all. If you want them to continue to do so, you can have a logical OR with a `|`. Otherwise you don't need a `Q()` object - it's only for complex queries